content/questions/actions/question-088.md

```diff
@@ -0,0 +1,13 @@
+---
+archetype: "questions"
+title: "Question 088"
+question: "What does the "matrix" keyword do in a GitHub Actions workflow?"
+draft: false
+---
+
+
+> https://docs.github.com/en/enterprise-cloud@latest/actions/using-jobs/using-a-matrix-for-your-jobs
+1. [x] Allows defining multiple environments for a single job
```

Comment:
```suggestion
1. [x] Allows defining multiple job configurations to run in parallel
```

Reply:
You are right, but environments is one of the configurations. However, I accept your suggestions as it more generic